docs/explanations/qrao_explanations.rst

```diff
@@ -0,0 +1,368 @@
+and it will not be the identity.
+
+Practically, this means that our commutation relationship will break and
+:math:`\text{cut}(m) \not= \text{Tr}\big(H^{(1)}F^{(3)}(m)\big)`
+
+In order to restore the commutation of our encoding with our objective
+function, we must introduce an additional constraint on the form of our
+problem Hamiltonian. Specifically, we must guarantee that decision
+variables which share an edge in our input graph :math:`G` are not
+assigned to the same qubit under our embedding :math:`F`
+
+.. math:: \forall e_{ij} \in E,\quad F^{(3)}(\dots,m_i,\dots,m_j,\dots) = F^{(3)}(\dots,m_i,\dots)\otimes F^{(3)}(\dots,m_j,\dots)
+
+In [1] this is accomplished by finding a coloring of the graph G such
+that no vertices with the same color share an edge, and then assigning
+variables to the same qubit only if they have the same color.
+
+Quantum Rounding Schemes
+------------------------
+
+Because the final solution we obtain for the relaxed problem
+:math:`\rho_\text{relax}` is unlikely to be in the image of :math:`F`,
+we need a strategy for mapping :math:`\rho_\text{relax}` to the image of
+:math:`F` so that we may extract a solution to our original problem.
+
+In [1] there are two strategies proposed for rounding
+:math:`\rho_\text{relax}` back to :math:`m \in \\{-1,1\\}^M`.
+
+Semi-deterministic Rounding
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A natural choice for extracting a solution is to use the results of
+Table :math:`2` and simply estimate
+:math:`\text{Tr}(P_{[i]}\rho_\text{relax})` for all :math:`i` in order
+to assign a value to each variable :math:`m_i`. The procedure described
+in Table :math:`2` was intended for use on states in the image of
+:math:`F`, however, we are now applying it to arbitrary input states.
+The practical consequence is we will no longer measure a value close to
+{:math:`\pm 1`}, {:math:`\pm \sqrt{2}`}, or {:math:`\pm \sqrt{3}`}, as
+we would expect for the :math:`(1,1,1)`, :math:`(2,1,p)`, and
+:math:`(3,1,p)` QRACs, respectively.
+
+We handle this by returning the sign of the expectation value, leading
+to the following rounding scheme.
+
+.. math::
+
+   m_i = \left\\{\begin{array}{rl}
+         +1 & \text{Tr}(P_{[i]}\rho_\text{relax}) > 0 \\
+         X \sim\\{-1,1\\} & \text{Tr}(P_{[i]}\rho_\text{relax}) = 0 \\
+         -1 & \text{Tr}(P_{[i]}\rho_\text{relax}) < 0
+         \end{array}\right.
+
+Where :math:`X` is a random variable which returns either -1 or 1 with
+equal probability.
+
+Notice that semi-deterministic rounding will faithfully recover :math:`m`
+from :math:`F(m)` with a failure probability that decreases
+exponentially with the number of shots used to estimate each
+:math:`\text{Tr}(P_{[i]}\rho_\text{relax})`
+
+Magic State Rounding
+~~~~~~~~~~~~~~~~~~~~
+
+.. figure:: aux_files/magic_state_rounding.svg
+   :align: center
+   :width: 100%
+
+   Figure 1: Three different encodings, the states and the measurement bases, of variables into a
```

Comment:
sphinx seems to introduce "Fig. 1".
```suggestion
   Three different encodings, the states and the measurement bases, of variables into a
```
![image](https://github.com/qiskit-community/qiskit-optimization/assets/31178928/cd9e1bc3-0895-448d-b0f2-ea366d5fc0ee)